pkg/config/config.go

```diff
@@ -17,6 +17,11 @@ type Links struct {
 	Confirm string `default:"http://localhost/confirm/%v?token=%v"`
 }
 
+// OAuth2 is google config for oauth2 with google
```

Comment:
Is this config only required for Google ?

Reply:
my bad, i'll change the comment, required for facebook too :)